packages/web/src/components/dragndrop/Draggable.tsx

```diff
@@ -54,8 +54,9 @@ export const Draggable = (props: DraggableProps) => {
       // it forces a site refresh
       // on drop
       if (!isFirefox()) {
-        dt.setData('text/uri-list', link)
-        dt.setData('text/plain', link)
+        const fullLink = window.location.origin + link
```

Comment:
this will break w/ the current default link including https://audius.co. either let's check to see if it already has it or maybe we should visit the callsites of <Draggable>

Reply:
yup, will look